CHANGELOG.rst

```diff
@@ -2,7 +2,39 @@
 cyclus Change Log
 =================
 
-.. current developments
+Since last release
+====================
+
+**Added:**
+
+* adding compatibility with cbc 2.10
```

Comment:
Should we put PR numbers on each entry?

Reply:
we did not in the past.
I can do it for the coming release, and set it as a expectation...

Reply:
I actually thought it is better to have the PR numbers, so I added them.